sync15-adapter/Cargo.toml

```diff
@@ -16,8 +16,8 @@ hyper = "0.12.10"
 log = "0.4.5"
 lazy_static = "1.0"
 base16 = "0.1.1"
-failure = "0.1.2"
-failure_derive = "0.1.2"
+failure = "0.1.3"
```

Comment:
I know I gave you this patch, but we might want to bump the other users of failure in this workspace at the same time.